pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -94,6 +97,32 @@ func (m *OSDHealthMonitor) Update(removeOSDsIfOUTAndSafeToRemove bool) {
 
 // checkOSDHealth takes action when needed if the OSDs are not healthy
 func (m *OSDHealthMonitor) checkOSDHealth() error {
+	err := m.checkByOSDDump()
```

Comment:
```suggestion
	err := m.checkOSDHealth()
```

Reply:
The suggested method name is same as of the enclosing method, so proceeded with the name `checkOSDDump()`.